### PR TITLE
[LLM Inference] support qwen2 a8w8c8 inference

### PR DESCRIPTION
**Pull request by ckl117**
### PR types

New features
### PR changes

Others
### Description

support qwen2 a8w8c8 inference

### CODE REVIEW

File: paddlenlp/experimental/transformers/qwen2/modeling.py [hunk 1]
```diff
@@ -313,39 +455,267 @@ def set_state_dict(self, state_dict):
...
+                    )
+                    weight_scale_json_path = os.path.join(
+                        self.quant_model_path, f"weight_scales_{self.config.tensor_parallel_rank}.json"
+                    )
+                act_scale_loader = ActScalesLoader(
+                    act_scale_json_path, act_scale_map_dict, num_of_layers=self.config.num_hidden_layers
+                )
+                self.transformer_block.act_scales = act_scale_loader.scale
+                print(f"weight_scale_json_path = {weight_scale_json_path}")
+                # print(f'weight_scale_map_dict = {weight_scale_map_dict}')
```
Comment (yuanlehome) [inline paddlenlp/experimental/transformers/qwen2/modeling.py]: 调试信息删掉

Comment (ckl117) [reply paddlenlp/experimental/transformers/qwen2/modeling.py]: done

**Review by yuanlehome — APPROVED** [review]

**Comment by DesmonDay** [comment]
建议后面增加相应的单测，保证功能。 @ckl117 @yuanlehome

**Review by DesmonDay — APPROVED** [review]
LGTM

**Pull request merged**